README.md

```diff
@@ -1,2 +1,62 @@
-# catalog-of-my--hings-app
-"Catalog of my things" is  a simple console app that allows users to manage collections of the things they own.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CATALOG OF MY THINGS APP
+
+> "Catalog of my things" is  a simple console app that allows users to manage collections of the things they own.
+
+Additional description about the project and its features.
+## Built With
+- ruby
+- rubocop
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Text Editor (VScode, Atom, Sublime text, etc)
+- Basic knowledge of ruby
+
+### Setup
+
+- clone repository using `git clone`
+- git clone `https://github.com/kinginthenorthcodez/catalog-of-my-things-app`
+- cd `catalog-of-my-things-app`.
+
+## Author 1
+👤 **Baseem Shams**
+
+- GitHub: [@basem909](https://github.com/basem909)
+- Twitter: [@ShamsBassem](https://twitter.com/ShamsBassem)
+- LinkedIn: [Bassem Abdelrahman](https://www.linkedin.com/in/bassem-shams-126a29134/)
+
+
+## Author 2
+👤 **Isaac Maqueen**
+
+- GitHub: [@Isaac Maqueen](https://github.com/kinginthenorthcodez/catalog-of-my-things-app)
+
+## Author 3
+👤 **Gedewon Hailegebrale**
+
+- GitHub: [@githubhandle](https://github.com/gedewon)
+- Twitter: [@twitterhandle](https://twitter.com/gedewon)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/gedewon)
```

Comment:
[OPTIONAL] It would be better if you used `@gedewon` instead of `@twitterhandle` and there are two Author 3. I know these details are not a huge thing, but this will make your repo look more professional for the recruiters. 😎